src/generic/datavgen.cpp

```diff
@@ -2084,11 +2084,61 @@ bool wxDataViewMainWindow::EnableDragSource( const wxDataFormat &format )
 
 bool wxDataViewMainWindow::EnableDropTarget( const wxDataFormat &format )
 {
-    m_dropFormat = format;
-    m_dropEnabled = format != wxDF_INVALID;
+    if (format == wxDF_INVALID)
+        return false;
+
+    wxDataObjectComposite *dataObject;
+    wxDataViewDropTarget *dropTarget = static_cast<wxDataViewDropTarget*>(GetDropTarget());
```

Comment:
```suggestion
    wxDropTarget* const dropTarget = GetDropTarget();
```
It doesn't look like we actually need the cast to the derived class, do we?

Reply:
Yes, we can do w/o casting.